functests/4_latency/latency.go

```diff
@@ -99,7 +99,20 @@ var _ = Describe("[performance] Latency Test", func() {
 		Expect(err).ToNot(HaveOccurred(), "error looking for the optional selector: %v", err)
 
 		Expect(workerRTNodes).ToNot(BeEmpty())
-		workerRTNode = &workerRTNodes[0]
+
+		//At least one worker node should have cpu.Allocatable greater than the quantity requested by each test, else skip the test
+		for _, wrkNode := range workerRTNodes {
+			allocatableCPU, _ := wrkNode.Status.Allocatable.Cpu().AsInt64()
+			Expect(err).ToNot(HaveOccurred(), "error getting allocatable cpus on the node: %v", err)
```

Comment:
Which error you're checking here?

Reply:
Actually, there is no error to check, I must have forgotten to remove this line. Thanks for catching that,I removed it